lasertagger/bert/tokenization.py

```diff
@@ -161,11 +161,15 @@ def whitespace_tokenize(text):
 class FullTokenizer(object):
   """Runs end-to-end tokenziation."""
 
-  def __init__(self, vocab_file, do_lower_case=True):
+  SPECIAL_SYMBOL_LIST = [".", ",", ";", "!", "?", ":", 
```

Comment:
Either define this in the global scope, or define it as a variable of the class: self.special_symbol_list = ...
https://google.github.io/styleguide/pyguide.html#25-global-variables

If this is internal to the module, add a '_' prefix.

Reply:
Done.